skeleton.css

```diff
@@ -18,7 +18,7 @@
 .container {
   position: relative;
   width: 100%;
-  max-width: 960px;
+  max-width: 900px;
   margin: 0 auto;
   padding: 0 0px;
```

Comment:
Consider adding a little side padding for widths less than 900px. It looks crowded on the smaller screens.